policies/cla.html

```diff
@@ -34,9 +34,19 @@
           </p>
 
           <p>
-          A submission is considered trivial if the author and all reviewers
-          agree that it is. In this case a CLA is not required and the
-          commit text should include "CLA: trivial" on a line by itself.
+	  A submission is trivial if it is considered trivial under copyright
+	  law. Since we are not lawyers, we place the bar for trivial
+	  contributions very high. Corrections of grammatical or
```

Comment:
Maybe add "For example":

```
contributions very high. For example, corrections of grammatical or
```

Reply:
I had for example there, removed it, put it back, removed it.

And now put it back.